pragma-sdk/pragma_sdk/onchain/constants.py

```diff
@@ -20,14 +20,16 @@
 
 RPC_URLS: Dict[Network, List[str]] = {
     "mainnet": [
-        "https://starknet-mainnet.public.blastapi.io/rpc/v0_7",
+        "https://starknet-mainnet.public.blastapi.io",
         "https://free-rpc.nethermind.io/mainnet-juno",
         "https://api.cartridge.gg/x/starknet/mainnet",
+        "https://mainnet-pragma.karnot.xyz",
```

Comment:
Mmmmmhhh do we want to reveal publicly this rpc? We should keep it for us

Reply:
too late ?

Reply:
but yes no worries its fine